docs/tutorials/how-to-write-contracts-that-talk-to-each-other.md

```diff
@@ -418,17 +415,44 @@ describe("CalculatorAPI", function() {
       };
 
       const result = await contract.calculate(params);
-      console.log(result);
-      expect(result.lastResult).toBe("100");
+      expect(result).toBe("100");
     });
   });
 });
+
+```
+
+Next we will need to update our `package.json` file to let our testing suite know we are using a .env file to store our `CONTRACT_NAME`.
+
+> In Gitpod's file explorer on the left
+> - Open your `package.json` file (near the bottom of the root directory)
+> - replace line 15 with the code below
+
+```js
+"jest": "yarn build && env-cmd -f ./neardev/dev-account.env jest test",
 ```
 
+> After that is complete
+> - Click **File** >> **Save All** to save your changes
+
+Now let's test it out!
+
+> In Gitpod
+> - click **Terminal** >> **New Terminal** 
+>
+> In the new tab that opens at the bottom of Gitpod
+> - type `yarn jest` in the command prompt
+
+You should see a successful test that looks something like this: 
+
+![Cross contract call Jest test](/docs/assets/cross-contract-call-jest-test.png)
+
```

Comment:
Add a `heads up` :
Remember when we took a note that the 'Contract Called' and the 'Contract Signer' were the same in the test environment? Notice that they are different now! Can you figure out what this means?